app/travis_scripts/deploy-docker.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Comment:
perhaps should `set -e` here so it stops immediately if any of these commands fail

Reply:
done